packages/@okta/vuepress-site/code/rest/index.md

```diff
@@ -1,94 +1,100 @@
+* Import that file into Postman by selecting **Import** from the **File** menu.
+* In the **Import** window, leave **Import File** selected and click **Choose Files**.
+* Browse to your download location, select the JSON file, and click **Open**.
+* Click **Import**.
 
 > **Note:** You can import and work with the rest of the Okta API using the link at the top of each API reference page,
 or see [all Postman collections](/docs/reference/postman-collections/). This tutorial only requires the Users API collection.
 
-## Send a Request
+## Send a request
 
-Once you've imported the Users API collection, and added your Okta org information to your environment, you're ready to send a request.
+After you've imported the Users API collection and added your Okta org information to your environment, you're ready to send a request.
 
-To make sure everything works, send a request to list all the users in your org:
+To make sure everything works, send a request to list all of the users in your org:
 
-1. Select the **Collections** tab in Postman and open the **Users (Okta API)** collection. Open the **List Users** folder, and select **(GET) List Users**. This loads the List Users request into Postman, ready to send.
+1. Select the **Collections** tab in Postman and expand the **Users (Okta API)** collection.
+1. Expand the **List Users** folder and select **List Users**. This loads the List Users request into Postman, ready to send.
 1. Click **Send**. The result pane automatically displays the results of your request:
-![GET List Users](/img/postman_response.png "GET List Users")
 
 If you receive an error, it's likely that one of the values in the environment isn't set correctly. Check the values and try again.
 
-Once you have completed this simple request, you're ready to explore the Okta API!
+Once you have completed this simple request, you're ready to explore the Okta API.
 
 ## Tips
 
 Now that you have a working collection, you can use the following tips to work more efficiently.
 
-### Find IDs for Okta API Requests
+### Find IDs for Okta API requests
 
-Your imported collections contain URLs and JSON request bodies that have sample data with variables such as `{userId}`.
-Replace URL and body variables with the IDs of the resources you wish to specify.
+Your imported collections contain URLs and JSON request bodies that have sample data with variables such as `{userId}`. You can replace URL and body variables with the IDs of the resources that you want to specify.
 
 To find an ID:
 
-1. List a resource collection with a search or filter. For example, list the users in your org, as you did in the previous section.
-![List Users Response](/img/postman_response2.png "List Users Response")
+1. To get a user's ID, for example, send a request to list the users in your org like you did in the previous section. Each user listed in the response has an ID:
```

Comment:
I think we need to keep that first sentence ("List a resource collection with a search or filter.") to retain the general idea of what is being done in the specific example.